packages/actor-query-operation-leftjoin/test/ActorQueryOperationLeftJoin-test.ts

```diff
@@ -6,7 +6,7 @@ import { ArrayIterator, UnionIterator } from 'asynciterator';
 import { DataFactory } from 'rdf-data-factory';
 import * as sparqlee from 'sparqlee';
 import { isExpressionError } from 'sparqlee';
-import { ActorQueryOperationLeftJoin } from '../lib/ActorQueryOperationLeftJoin';
+import { ActorQueryOperationLeftJoin } from '../lib';
```

Comment:
This change also shouldn't be needed.

Reply:
gtk, webstorm suggested it, otherwise i wouldn't have done this.